drivers/ieee802154/ieee802154_cc2520.c

```diff
@@ -973,6 +963,81 @@ static int power_on_and_setup(struct device *dev)
 	return 0;
 }
 
+static struct cc2520_gpio_configuration *configure_gpios(struct device *dev)
+{
+	struct cc2520_context *cc2520 = dev->driver_data;
+	struct device *gpio;
+
+	/* VREG_EN */
+	gpio = device_get_binding(DT_INST_0_TI_CC2520_VREG_EN_GPIOS_CONTROLLER);
+	if (!gpio) {
+		return NULL;
+	}
+
+	cc2520->gpios[CC2520_GPIO_IDX_VREG_EN].pin = DT_INST_0_TI_CC2520_VREG_EN_GPIOS_PIN;
+	gpio_pin_configure(gpio, cc2520->gpios[CC2520_GPIO_IDX_VREG_EN].pin,
+			   GPIO_OUTPUT | DT_INST_0_TI_CC2520_VREG_EN_GPIOS_FLAGS);
```

Comment:
Let's initialize all outputs to a known value.

Reply:
Low seems to be the off/in-reset state so I went with that.